src/core/remixes/convenience/UIView+RMXRemixerAPI.h

```diff
@@ -0,0 +1,60 @@
+/*
+ Copyright 2016-present Google Inc. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import <UIKit/UIKit.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+/** Category for UIView that adds a convenience API for Remixer. */
+@interface UIView (RMXRemixerAPI)
+
+/**
+ Creates a RangeVariable with minValue 0 and maxValue 1.
+ It automatically updates the view's property when the selectedValue changes.
+
+ @return The current value of the property.
+ */
+- (CGFloat)alphaVariableForKey:(NSString *)key updateProperty:(NSString *)property;
```

Comment:
Should this be rangeVariable?

Or another question to answer in the method's docs here: "What's an alpha variable?"